packages/language-server/test/completion-items-spec.ts

```diff
@@ -170,24 +180,52 @@ describe("the UI5 language assistant Code Completion Services", () => {
       {
         label: "BusyDialog",
         tagName: "m:BusyDialog",
+        firstAttribute: `xmlns:m="sap.m"`,
         replacedText: "sap.m.Busy"
       },
       {
         label: "BusyIndicator",
         tagName: "m:BusyIndicator",
+        firstAttribute: `xmlns:m="sap.m"`,
         replacedText: "sap.m.Busy"
       }
     ]);
 
     expect(suggestionKinds).to.deep.equal([CompletionItemKind.Class]);
   });
 
+  it("will insert class namespace with a new name when another namespace with the short name is defined", () => {
```

Comment:
the complexity of what this test does definitively warrants snapshot testing (in the future)